### PR TITLE
Add paragraph about district partnership

### PR DESCRIPTION
**Pull request by bethanyaconnor**
I missed this content in my first pass at updating so adding it now. See the [spec](https://docs.google.com/document/d/1HX7n1stbdmzzWrUi8--Z-6GuCRUqyzoNkZbH9w7TO7I/edit#heading=h.2dl2k9kap0g1) for more info.

Email preview:
Screenshot 2024-01-10 at 11 44 52 AM

### CODE REVIEW

File: dashboard/app/views/pd/application/teacher_application_mailer/admin_approval_completed.html.haml [hunk 1]
```diff
@@ -13,6 +13,16 @@
   = link_to 'code.org/apply', 'https://code.org/apply'
   or reach out to me directly for additional information.
 
+%p
+  Districts can now partner with Code.org directly, at no cost, to get
+  support in expanding computer science for all students. Participating
+  districts will receive high-quality, open-source K-12 computer science
+  curriculum and support such as implementation guidance and access to
+  local & national communities of practice. More information
```
Comment (kelbyhawn) [inline dashboard/app/views/pd/application/teacher_application_mailer/admin_approval_completed.html.haml]: This looks good, but I have one suggestion if the O&A team is ok with it. This makes it a complete sentence and the links are phrases which is better for accessibility.

P.s. how can I view this file (or email files in general) locally?

```suggestion
  local & national communities of practice.
  = link_to('Get more information', 'https://code.org/districts#email-signup-form') + ','
  or
  = link_to('enroll here', 'https://docs.google.com/forms/d/e/1FAIpQLSfFV_iJoDK4erZxIeX_tZiq5vZ7ZDa9HFjHbYzBDNLT9rIFhQ/viewform?usp=sf_link') + '.'
```

Comment (TurnerRiley) [reply dashboard/app/views/pd/application/teacher_application_mailer/admin_approval_completed.html.haml]: You can see the list of emails locally at http://localhost-studio.code.org:3000/rails/mailers and specifically this one at http://localhost-studio.code.org:3000/rails/mailers/pd_teacher_application_mailer/admin_approval_completed__with_partner

Comment (kelbyhawn) [reply dashboard/app/views/pd/application/teacher_application_mailer/admin_approval_completed.html.haml]: Thanks @TurnerRiley! 

Here's a screenshot:

Screenshot 2024-01-10 at 10 08 36 AM

Comment (bethanyaconnor) [reply dashboard/app/views/pd/application/teacher_application_mailer/admin_approval_completed.html.haml]: That sounds fine to me but I'll admit I don't have strong opinions here. @dmcavoy any thoughts?

**Pull request merged**